lib/pure/sugar.nim

```diff
@@ -335,6 +335,20 @@ macro collect*(init, body: untyped): untyped {.since: (1, 1).} =
       call.add init[i]
   result = newTree(nnkStmtListExpr, newVarStmt(res, call), resBody, res)
 
+since (1, 3):
```

Comment:
This should be applied as a pragma to the template instead.

Reply:
Alas, it currently cannot because we don't have custom pragmas on template declarations.